src/cli/install.ps1

```diff
@@ -1,176 +1,220 @@
-    Write-Error $_
-    exit 1
-  }
-  Move-Item "${BunBin}\$Target\bun.exe" "${BunBin}\bun.exe" -Force
-
-  Remove-Item "${BunBin}\$Target" -Recurse -Force
-  Remove-Item $ZipPath -Force
-
-  $BunRevision = "$(& "${BunBin}\bun.exe" --revision)"
-  if ($LASTEXITCODE -eq 1073741795) { # STATUS_ILLEGAL_INSTRUCTION
-    if ($IsBaseline) {
-      Write-Output "Install Failed - bun.exe (baseline) is not compatible with your CPU.`n"
-      Write-Output "Please open a GitHub issue with your CPU model:`nhttps://github.com/oven-sh/bun/issues/new/choose`n"
-      exit 1
-    }
+<#
+.SYNOPSIS
 
-    Write-Output "Install Failed - bun.exe is not compatible with your CPU. This should have been detected before downloading.`n"
-    Write-Output "Attempting to download bun.exe (baseline) instead.`n"
-
-    Install-Bun -Version $Version -ForceBaseline $True
-    exit 1
-  }
-  if ($LASTEXITCODE -eq 3221225781) { # STATUS_DLL_NOT_FOUND
-    Write-Output "Install Failed - You are missing a DLL required to run bun.exe"
-    Write-Output "The command '${BunBin}\bun.exe --revision' exited with code ${LASTEXITCODE}`n"
-    exit 1
-  }
-  if ($LASTEXITCODE -ne 0) {
-    Write-Output "Install Failed - could not verify bun.exe"
-    Write-Output "The command '${BunBin}\bun.exe --revision' exited with code ${LASTEXITCODE}`n"
-    exit 1
-  }
-  $DisplayVersion = if ($BunRevision -like "*-canary.*") {
-    "${BunRevision}"
-  } else {
-    "$(& "${BunBin}\bun.exe" --version)"
-  }
-
-  $C_RESET = [char]27 + "[0m"
-  $C_GREEN = [char]27 + "[1;32m"
-
-  try {
-    $null = New-Item -ItemType HardLink -Path "${BunBin}\bunx.exe" -Target "${BunBin}\bun.exe" -Force
-  } catch {
-    Write-Warning "Could not create a hard link for bunx, falling back to a cmd script`n"
-    Set-Content -Path "${BunBin}\bunx.cmd" -Value "@%~dp0bun.exe x %*"
-  }
+A script to install bun.
+
+.DESCRIPTION
+
+This script installs bun. By default, the installation directory is %LOCALAPPDATA%\Programs\bun.
+The installation directory can be customized by specifying the custom installation path through the %BUN_INSTALL% environment variable.
+To install bun to %USERPROFILE%\.bun, simply use the -InstallToUserProfile switch.
+
+.PARAMETER Release
+Changes the version from "Release" (default, latest stable version) to "Canary" (latest experimental build).
+
+.PARAMETER ForceBaseline
+Enforces the installation of the baseline variant that accommodates older CPUs. If the CPU is not supported, the baseline variant is installed automatically.
+
+.INPUTS
+None.
 
-  Write-Output "${C_GREEN}Bun ${DisplayVersion} was installed successfully!${C_RESET}"
-  Write-Output "The binary is located at ${BunBin}\bun.exe`n"
+.OUTPUTS
+None.
 
-  Write-Warning "Bun for Windows is currently experimental.`nFor a more stable experience, please install Bun within WSL:`nhttps://bun.sh/docs/installation`n"
+.LINK
+https://bun.sh/docs/installation
+#>
+#Requires -Version 7
```

Comment:
This will not work on powershell 5. the original script was designed to work on that. Windows 10 Home does not ship with PowerShell 7, so it will not work on a fresh machine.